README.md

```diff
@@ -1,70 +1,29 @@
 
 # jpo-security-svcs
-This module exposes a RESTful API for performing cryptographic functions. The following paths identify the functions:
-|Verb|path|Content Type|Functionality|Request Body Format|Response Body Format|
-|--|--|--|--|--|--|
-|POST|/sign|application/json|signs data provided in the body of the request|{"message":"Base64 encoded unsigned data"}|{"result": "Base64 Encoded Signed Data"}
+This module exposes a RESTful API for performing cryptographic signing of messages.
 
+Cryptographic signing is carried out against a remote instance, which requires mutual TLS authentication to secure the communications. This requires using private and public keys to secure the communications between this instance and the remote signing instance.
```

Comment:
If you want to make the repo sound real fancy, you can call it what it is - TLS using asymmetric cryptography to secure instance communication. Nothing wrong with explaining how it works, though.